modules/vertice-governance-role/variables.tf

```diff
@@ -58,6 +58,8 @@ variable "billing_policy_addons" {
     es_ri          = optional(bool, true),
     rds_ri         = optional(bool, true),
     redshift_ri    = optional(bool, true),
+    # why also here?
```

Comment:
The variable needs to propagate to the `modules/vertice-governance-role` submodule, which is called from the root module, hence both places need the variable.